DisguiseUnityRenderStream/Runtime/DisguiseRenderStreamSettings.cs

```diff
@@ -7,12 +7,26 @@ class DisguiseRenderStreamSettings : ScriptableObject
 {
     public enum SceneControl
     {
+        /// <summary>
+        /// Restricts the disguise software's control of scenes and instead merges all channels and remote parameters into a single scene.
+        /// </summary>
         Manual,
+        
+        /// <summary>
+        /// Allows scenes to be controlled from inside the disguise software; channels are merged into a single list (duplicates removed) and remote parameters are per-scene.
+        /// </summary>
         Selection
     }
 
     [SerializeField]
-    public SceneControl sceneControl;
+    public SceneControl sceneControl = SceneControl.Manual;
+    
+    /// <summary>
+    /// When true, the Unity window will be able to display streams or live textures to the local screen.
+    /// The generated schema will include remote parameters to select the texture to display and how to resize it to fit the screen.
+    /// </summary>
+    [SerializeField]
+    public bool exposePresenter = true;
```

Comment:
Minor: If we keep the default UI for those those two fields (which on first sight are sufficient), maybe we should add tooltips with parts of the summary for those two parameters?